src/Symfony/Component/HttpFoundation/Tests/ResponseFunctionalTest.php

```diff
@@ -42,7 +42,7 @@ public static function tearDownAfterClass(): void
      */
     public function testCookie($fixture)
     {
-        if (\PHP_VERSION_ID >= 80000 && 'cookie_max_age' === $fixture) {
+        if ('cookie_max_age' === $fixture) {
```

Comment:
Please delete the corresponding fixture instead.

Reply:
good catch!